lib/ams_lazy_relationships.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 require "batch-loader"
+
+# That is a missing dependency of AMS v0.10.0.rc4 in fact (
+require "active_support/core_ext/string/inflections"
```

Comment:
Are you sure this is required? I think people using rc4 should take care of requiring it themselves. Our gem is just an add-on to that. 

We can require this file in specs if they're not passing though.

What do you think?

Reply:
agreed, on my way )

Reply:
Some thoughts:

1) it's good to require dependencies before the using of them, because, well, the things will not be rolling otherwise ) . Our lib files have a references to `::ActiveModel::Serializer` but never required it in fact. That was just a lucky case that most part of host applications required `active_model_serializers` beforehand, but technically our gem was corrupted.
2) the same related to AMS v0.10.0.rc4 - it is not able to be loaded is it is due to the corrupted dependencies. As a result our gem becomes to be corrupted because we relying on AMS and want to require it at the beginning
3) Don't think that it's a good way to try fixing the nested dependencies (active_support missed by AMS), that looks like a fix of floor curvature by opposite curvature of furniture foot ) Better to get it fixed in upstream, say, in v0.10.0
4) but sure I agree to keep in attention rc4 if you use it ) As for me, I feel the same passion to v0.10.3 ))